lib/console.js

```diff
@@ -85,7 +85,10 @@ Console.prototype.trace = function trace() {
 
 Console.prototype.assert = function(expression) {
   if (!expression) {
-    var arr = Array.prototype.slice.call(arguments, 1);
+    var arr = [];
```

Comment:
I thought it might look a little messy since there would need to be a couple `- 1`s since the first item is being sliced off, but I agree it would be better to make it more similar to the other instances.